samples/scripts/diagnosticsetting.fsx

```diff
@@ -0,0 +1,42 @@
+#r "./libs/Newtonsoft.Json.dll"
+#r "../../src/Farmer/bin/Debug/netstandard2.0/Farmer.dll"
+open Farmer
+open Farmer.Builders
+open System
+
+let storageAccountResourceType=ResourceType("Microsoft.Storage/storageAccounts","")
```

Comment:
This shouldn't be required - it exists in the `Arm` namespace.

Reply:
lol, I forgot that they already exist :joy: